lms/static/js/spec/edxnotes/plugins/accessibility_spec.js

```diff
@@ -37,7 +37,9 @@ define([
         });
 
         afterEach(function () {
-            _.invoke(Annotator._instances, 'destroy');
+            while (Annotator._instances.length > 0) {
```

Comment:
Ah I see. A side effect of `destroy` is to remove it from `Annotator._instances`. I thought we were somehow doing that manually. :+1: